packages/ui/src/utils/UseVisibleOrFilterableField.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from "react";
+
+export const useVisibleOrFilterableField = <T extends string>(
```

Comment:
Maybe we can change this to more generic name eg `useColumnsMap`? :thinking:

Reply:
sounds good.